pkg/image/definition.go

```diff
@@ -48,6 +49,28 @@ type Suma struct {
 	GetSSL        bool   `yaml:"getSSL"`
 }
 
+type Hauler struct {
+	ContainerImages []ContainerImage `yaml:"images"`
+	HelmCharts      []HelmChart      `yaml:"charts"`
+	Files           []File           `yaml:"files"`
+}
+
+type ContainerImage struct {
+	Name string `yaml:"name"`
+	Key  string `yaml:"key"`
```

Comment:
What type of key do we specify here?

Reply:
Putting what we discussed here for visibility:
https://rancherfederal.github.io/hauler-docs/docs/introduction/quickstart#using-the-command-line
This key is for supply chain artifacts as shown in the link above.
Will rename `Key` to `SupplyChainKey`